src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -0,0 +1,216 @@
+                        styles.mr3,
+                        onboardingIsMediumOrLargerScreenWidth ? styles.ml3 : styles.ml0,
+                        styles.onboardingIconWrapper,
+                    ]}
+                />
+            ),
+            rightElement: onboardingIsMediumOrLargerScreenWidth ? <View style={styles.mr3} /> : null,
+            isSelected: userReportedIntegration === null,
+        };
+        return [...policyAccountingOptions, noneAccountingOption];
+    }, [StyleUtils, onboardingIsMediumOrLargerScreenWidth, styles.ml0, styles.ml3, styles.mr3, styles.onboardingIconWrapper, theme.success, translate, userReportedIntegration]);
+
+    const footerContent = (
+        <>
+            {!!error && (
+                <FormHelpMessage
+                    style={[styles.ph1, styles.mb2]}
+                    isError
+                    message={error}
+                />
+            )}
+            <Button
+                success
+                large
+                text={translate('common.confirm')}
+                // eslint-disable-next-line rulesdir/prefer-early-return
+                onPress={() => {
+                    if (userReportedIntegration === undefined) {
+                        setError(translate('onboarding.purpose.errorSelection'));
+                        // eslint-disable-next-line no-useless-return
+                        return;
+                    }
+
+                    if (!onboardingPurposeSelected) {
+                        return;
+                    }
+
+                    if (onboardingPolicyID) {
+                        Policy.enablePolicyConnections(onboardingPolicyID, true);
+                    }
+
+                    Report.completeOnboarding(
+                        onboardingPurposeSelected,
+                        CONST.ONBOARDING_MESSAGES[onboardingPurposeSelected],
+                        undefined,
+                        undefined,
+                        onboardingAdminsChatReportID ?? undefined,
+                        onboardingPolicyID,
+                        undefined,
+                        onboardingCompanySize,
+                        userReportedIntegration,
+                    );
+
+                    Welcome.setOnboardingAdminsChatReportID();
+                    Welcome.setOnboardingPolicyID();
+
+                    Navigation.dismissModal();
+
+                    // Only navigate to concierge chat when central pane is visible
+                    // Otherwise stay on the chats screen.
+                    if (!shouldUseNarrowLayout && !route.params?.backTo) {
+                        Report.navigateToConciergeChat();
+                    }
+                }}
+                pressOnEnter
+            />
+        </>
+    );
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            shouldEnableMaxHeight
+            shouldEnableKeyboardAvoidingView
```

Comment:
Do we need these? We don't have any inputs that require keyboard

Reply:
No need